app/assets/javascripts/wizard/controllers/application.js

```diff
@@ -16,7 +16,7 @@ export default Controller.extend({
       return [];
     }
 
-    const fontField = fontsStep.get("fieldsById.font_previews");
-    return fontField.choices.map((choice) => `font-${choice.data.class}`);
+    const fontField = fontsStep.get("fieldsById.body_font");
+    return fontField.choices.map((choice) => `body_font-${choice.id}`);
```

Comment:
It's a bit odd to mix `_` and `-` in the id here. If it's not too much work I would really prefer if we stuck to dashes only for front end stuff.